conference.js

```diff
@@ -1950,9 +1937,6 @@ export default {
                     }));
             });
 
-        APP.UI.addListener(UIEvents.NICKNAME_CHANGED,
-            this.changeLocalDisplayName.bind(this));
```

Comment:
Ditto

Reply:
The function is converted to action and it is gone from conference.js